bottom-bar/src/main/java/com/roughike/bottombar/BottomBarBadge.java

```diff
@@ -203,8 +203,11 @@ private void adjustPositionAndSize(View tabToAddTo) {
         setTranslationY(10);
 
         int size = Math.max(getWidth(), getHeight());
-        getLayoutParams().width = size;
-        getLayoutParams().height = size;
+
+        ViewGroup.LayoutParams params = getLayoutParams();
+        params.width = size;
+        params.height = size;
+        setLayoutParams(params);
```

Comment:
Maybe in the `onGlobalLayout` callback we should unregister it and call `invalidate()` right after calling `adjustPositionAndSize`